test/other/metadce/test_metadce_files_wasmfs.size

```diff
@@ -1 +1 @@
-52341
+55829
```

Comment:
I looked into this code size regression as it surprised me. What seems to happen is that even though we don't call `wasmfs_flush`, and it will never be called since the default (and what is checked in the metadce test) is no-exit-runtime mode, we do actually include that code. It appears since there is an atexit call pointing to that function. That atexit does nothing in no-exit-runtime mode, but we'd need LTO to notice that (testing with LTO I don't see it removed there either, but I think that's another issue).

That's a more general issue than WasmFS so we don't need to block this PR on it, though.

Reply:
That's pretty interesting. From what I understand in the issue, does this issue exist with all destructors currently in WasmFS?

Reply:
Yeah... I spent a bunch of time investigating this in https://github.com/emscripten-core/emscripten/issues/19993 - really unfortunate, any (global) destructor is kept alive right now. Hopefully we find a way to fix that!